### ISSUE TITLE
Kirigami Demo App launches, but some widgets seem overly large compared to others

### ISSUE THREAD

**Issue by omac777** [ubports/ubuntu-touch#911]
Nexus 5
ubuntu touch 16.04 2018-10-13

The main gui menu buttons seems huge, but when I traverse to list all the different demos within, the listed items have a seemingly correct/legible font size and item-real-estate size in terms of width and height and aspect ratio of screen real-estate.

**Comment by omac777** [comment]
Both the Kirigami Demo App and the QQC2 Gallery App are both behaving correctly as of Release Candidate Channel Version 19 and Development Channel version 674+(most recent 679).Thank you for listening.

    On Tuesday, October 30, 2018, 7:24:14 p.m. EDT, Florian Leeber <notifications@github.com> wrote:  
 
 
Dalton, shouldnt this work already properly?

**Comment by Flohack74** [comment]
Actually where did you get the Kirigami app?

**Comment by doniks** [comment]
@Flohack74 It's in the store

**Comment by omac777** [comment]
I got it through the ubuntu touch installer app "OpenStore". Both the Kirigami Demo and the QQC2 Gallery demos are behaving very well and very quick to respond to user touch actions!!! I cannot emphasize enough my deep satisfaction for ubuntu touch on the nexus 5 now.  I wish there were more  RAM/cores/storage in order to develop apps without any issues on it.  When using the rust on it, I got an LLVM out of memory error on it.  One app I was compiling with it took 10 minutes to compile so I am motivated to see ubuntu touch on more capable phones.  When I saw the spec for the fairphone being 2GB of RAM I was disappointed.

The following phones have 6 to 8 cores with 6 GB RAM to 10 GB RAM and crazy amazing cameras on them:

Archos Diamond Omega - Full phone specifications Oppo Realme 2 Pro - Full phone specifications Asus ROG Phone - Full phone specifications Sony Xperia XZ3 - Full phone specifications HTC Exodus 1 - Full phone specifications
ZTE nubia X - Full phone specifications 
OnePlus 6T - Full phone specifications
Lenovo Z5 Pro - Full phone specifications
LG V40 ThinQ - Full phone specifications
Xiaomi Mi Mix 3 - Full phone specifications
Samsung Galaxy Note9 - Full phone specifications Google Pixel 3 XL - Full phone specifications Huawei Mate 20 Pro - Full phone specifications Xiaomi Pocophone F1 - Full phone specifications 
    On Friday, November 2, 2018, 8:07:04 p.m. EDT, Florian Leeber <notifications@github.com> wrote:  
 
 
Actually where did you get the Kirigami app?
